functional-tests/src/test/groovy/FrontendMinespace/spec/PublicFrontendLogin.groovy

```diff
@@ -0,0 +1,33 @@
+package spec
+
+import geb.spock.GebReportingSpec
+import spock.lang.*
+
+import publicFrontend.pages.PublicFrontendLoginPage
+import publicFrontend.pages.KeycloakLoginPage
+import publicFrontend.pages.HomePage
+
+import utils.Const
+
+@Title("Public-Frontend-LoginPage")
```

Comment:
Same with the namespace standardization here.

Reply:
refactored